grafana_arena_dashboard.json

```diff
@@ -1261,30 +1257,30 @@
       "pluginVersion": "12.0.0",
       "targets": [
         {
-          "datasource": {
-            "type": "prometheus",
-            "uid": "${DS_PROMETHEUS}"
-          },
           "disableTextWrap": false,
-          "editorMode": "builder",
-          "expr": "avg by(le) (arena_game_tick_duration_bucket)",
+          "editorMode": "code",
+          "expr": "rate(game_updater_broadcast_binary_size_sum[5m]) / rate(game_updater_broadcast_binary_size_count[5m]) * 30\n",
```

Comment:
based on the name of the panel and its description I would expect this to be the estimate of the size of a single update, not an estimate of the amount of data sent per second (to each client)

Reply:
Ugh the per second thing is already in the panel. What name do you suggest?
<img width="591" alt="Screenshot 2025-05-13 at 6 22 57 PM" src="https://github.com/user-attachments/assets/f2657817-73c9-475c-8ea9-916cf87962c8" />

Reply:
The data is an estimate bc we multiply by 30 (at least that's what I assumed from the code) the binary size of an update (assuming we always reach 30 updates per second) so I would mention that it's an estimate either in the name or description.

Alternatively, we can just emit the event on each update and show a last value of that (or the buckets one)

Reply:
> Alternatively, we can just emit the event on each update and show a last value of that (or the buckets one)
If you do this, you only get 1 value while there are N >=0 active games.